File: call-management/rule-management/update-call-recording-settings/code-samples/updateCallRecordingSettings.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'onDemand' => array(
        'enabled' => true
    ),
    'automatic' => array(
        'enabled' => true,
        'outboundCallTones' => true,
        'outboundCallAnnouncement' => true,
        'allowMute' => true,
        'extensionCount' => 000
    ),
    'greetings' => array(
        array(
            'type' => 'StartRecording',
            'mode' => 'Default'
        )  
    )
)

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->put("/restapi/v1.0/account/{$accountId}/call-recording", $body);
?>

File: call-management/rule-management/update-call-recording-settings/code-samples/updateCallRecordingSettings.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string accountId = "<ENTER VALUE>";

// POST BODY
CallRecordingSettingsResource callRecordingSettingsResource = new CallRecordingSettingsResource {
    onDemand = new OnDemandResource {
        enabled = true
    },
    automatic = new AutomaticRecordingResource {
        enabled = true,
        outboundCallTones = true,
        outboundCallAnnouncement = true,
        allowMute = true,
        extensionCount = 000
    },
    greetings = new[] {
        new GreetingResource {
            type = 'StartRecording',
            mode = 'Default'
        },
    }
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Account(accountId).CallRecording().Put(callRecordingSettingsResource);
// PROCESS RESPONSE
